File: Curso_logica_de_programacao/Exercicios_logica_em_C#/Idades/Idades.cs
using System.Globalization;

namespace Idades;

class Program
{
    static void Main(string[] args)
    {

        CultureInfo CI = CultureInfo.InvariantCulture;

        string nome1, nome2;
        int idade1, idade2;

        Console.WriteLine("Dados da primeira pessoa:");
        Console.Write("Nome: ");
        nome1 = Console.ReadLine();

        Console.Write("Idade: ");
        idade1 = int.Parse(Console.ReadLine());

        Console.WriteLine("Dados da segunda pessoa: ");
        Console.Write("Nome: ");
        nome2 = Console.ReadLine();

        Console.Write("Idade: ");
        idade2 = int.Parse(Console.ReadLine());

        double media_idade = (double)(idade1 + idade2) / 2.0;

        System.Console.WriteLine("A idade media de " + nome1 + " e " + nome2 + " e de " + media_idade.ToString("F1", CI) + " anos");
    }
}

File: Algoritmos_e_Logica_de_Programacao/Exercícios_logica_em_Python/Idades/Idades.py
Nome1: str
Nome2: str
idade1: int
idade2: int


print("Dados da primeira pessoa: ")
Nome1 = input("Nome: ")
idade1 = int(input("Idade: "))

print("Dados da segunda pessoa: ")
Nome2 = input("Nome: ")
idade2 = int(input("Idade: "))

media: float
media = (idade1 + idade2) / 2

print(f"A idade media de {Nome1} e {Nome2} e de {media} anos")